.idea/gradle.xml

```diff
@@ -1,14 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="GradleMigrationSettings" migrationVersion="1" />
```

Comment:
this file should be not get committed

Reply:
When open with Android Studio preview it always change that file.

The .gitignore doesn't prevent the file to accidentally commit too. 
Because that file already exists in the repository. 

So can we try remove this file from repository first (In another PR). 
Then we can comeback and see that the .gitingore help us or not in this PR

Reply:
> So can we try remove this file from repository first (In another PR).
ah ya, this make sense.

Reply:
@wasinpp do rebase develop. i made an update https://github.com/radityagumay/QualityMatters/commit/3e65e579149e1afa8ea6290fe6a8a870f5400981

Reply:
Done rebase on force push to this branch mas.